packages/lesswrong/lib/collections/localgroups/schema.ts

```diff
@@ -96,6 +96,7 @@ const schema: SchemaType<DbLocalgroup> = {
       needsUpdate: data => ('googleLocation' in data),
       getValue: async (localgroup) => {
         if (localgroup.googleLocation) return googleLocationToMongoLocation(localgroup.googleLocation)
+        return null
```

Comment:
This was needed in order for clearing the location input to also clear the `mongoLocation` field